docs/src/partials/event-webhook/_event-webhook-list.md

```diff
@@ -893,6 +893,9 @@ Triggered when an experiment is created
             banditScheduleUnit?: ("days" | "hours") | undefined;
             banditBurnInValue?: number | undefined;
             banditBurnInUnit?: ("days" | "hours") | undefined;
+            linkedFeatures?: string[] | undefined;
+            hasVisualChangesets?: boolean | undefined;
+            hasURLRedirects?: boolean | undefined;
```

Comment:
Unrelated to these changes, but we should mark this file as generated in `.gitattributes` so it gets collapsed during PR reviews.